libs/hdf-converters/sample_jsons/sarif_mapper/sarif-hdf.json

```diff
@@ -1,981 +1,1499 @@
-                            "CWE-120"
-                        ],
-                        "nist": [
-                            "SI-10"
-                        ]
-                    },
-                    "descriptions": [],
-                    "refs": [],
-                    "source_location": {
-                        "ref": "test/test-patched.c",
-                        "line": 58
-                    },
-                    "title": "buffer/_tcsncpy",
-                    "id": "FF1009",
-                    "desc": "Easily used incorrectly; doesn't always \\0-terminate or check for invalid pointers [MS-banned] (CWE-120).",
-                    "impact": 0.3,
-                    "code": "",
-                    "results": [
-                        {
-                            "status": "failed",
-                            "code_desc": "URL : test/test-patched.c LINE : 58 COLUMN : 3",
-                            "run_time": 0,
-                            "start_time": ""
-                        },
-                        {
-                            "status": "failed",
-                            "code_desc": "URL : test/test.c LINE : 58 COLUMN : 3",
-                            "run_time": 0,
-                            "start_time": ""
-                        }
-                    ]
-                },
-                {
-                    "tags": {
-                        "cwe": [
-                            "CWE-126"
-                        ],
-                        "nist": [
-                            "SI-10"
-                        ]
-                    },
-                    "descriptions": [],
-                    "refs": [],
-                    "source_location": {
-                        "ref": "test/test-patched.c",
-                        "line": 62
-                    },
-                    "title": "buffer/strlen",
-                    "id": "FF1022",
-                    "desc": "Does not handle strings that are not \\0-terminated; if given one it may perform an over-read (it could cause a crash if unprotected) (CWE-126).",
-                    "impact": 0.3,
-                    "code": "",
-                    "results": [
-                        {
-                            "status": "failed",
-                            "code_desc": "URL : test/test-patched.c LINE : 62 COLUMN : 7",
-                            "run_time": 0,
-                            "start_time": ""
-                        },
-                        {
-                            "status": "failed",
-                            "code_desc": "URL : test/test.c LINE : 62 COLUMN : 7",
-                            "run_time": 0,
-                            "start_time": ""
-                        }
-                    ]
-                }
-            ],
-            "sha256": "407f6e03f5e117392a5629be0d3fa1af036b4bd09f33a5e3159d9faf1f7381e8"
-        }
-    ]
+	"platform": {
```

Comment:
Can we be consistent with our indentation level? I believe `JSON.stringify(object, null, 2)` is what we normally use